scripts/test

```diff
@@ -89,7 +89,7 @@ echo Running: build-token-generator
 path=$(pwd)
 ./tests/v2/integration/bin/integrationsetup
 export CATTLE_TEST_CONFIG=${path}/config.yaml
-go test -v ./tests/v2/integration/...
+go test -timeout 60m -v ./tests/v2/integration/...
```

Comment:
If this runs longer than a few minutes, we know it's broken right? Should we adjust the timeout to a few minutes? Or was there another reason to make it 60?

Reply:
I forgot to remove this, I tried this out for a debugging. I'll remove it